docs/organizations/managing-repositories.md

```diff
@@ -11,7 +11,7 @@ This page lists the repositories that already belong to your organization on Cod
 -   Complexity metric
 -   Duplication metric
 -   Coverage metric
--   Last updated time
+-   Date of the last commit
```

Comment:
I'm guessing that this is actually the date of the last commit **in one of the enabled branches on Codacy**. Can anyone confirm this so we can also add this extra detail?

Reply:
In lieu of the newest information, I now believe that the date is of the last commit pushed to **any** branch.